{"result": "error", "start_at": "2014-05-16 11:34:21.442626", "log": "report/2014-05-16_11:34:21.440443/error/Failure.runTest@2014-05-16_11:34:21.442626/log.txt", "trace": ["Traceback (most recent call last):\n", "  File \"/usr/lib/python2.7/unittest/case.py\", line 327, in run\n    testMethod()\n", "  File \"/usr/local/lib/python2.7/dist-packages/nose/loader.py\", line 402, in loadTestsFromName\n    module = resolve_name(addr.module)\n", "  File \"/usr/local/lib/python2.7/dist-packages/nose/util.py\", line 311, in resolve_name\n    module = __import__('.'.join(parts_copy))\n", "ImportError: No module named stress\n"], "screenshot_at_failure": "report/2014-05-16_11:34:21.440443/error/Failure.runTest@2014-05-16_11:34:21.442626/failure.png", "end_at": "2014-05-16 11:34:23.190760", "expect": null, "name": "Failure.runTest"}
